src/lib.rs

```diff
@@ -1767,6 +1775,35 @@ impl<K: KVStore + Sync + Send + 'static> Node<K> {
 	pub fn verify_signature(&self, msg: &[u8], sig: &str, pkey: &PublicKey) -> bool {
 		self.keys_manager.verify_signature(msg, sig, pkey)
 	}
+
+	pub fn reset_scorer_state(&self) -> Result<(), Error> {
```

Comment:
minor: it's resetting more than the scorer. Mutiny calls it `reset_router` which I think makes a bit more sense. Should we call it that?

Reply:
Sure, I'll fix shortly